app/controllers/search_controller.rb

```diff
@@ -0,0 +1,22 @@
+class SearchController < ApplicationController
+  def index
+    search_ro = search_params
```

Comment:
Searchコントローラにはトレンドの取得とかで別の関数を後々実装すると思うんだけど、そのときはsearch_param関数を使わない気がします
コントローラ全体で汎用なものでも無いので、普通にここで
```suggestion
    search_ro = SearchRo.new(params)
```
として取得した方が可読性が高いと思う